src/network/network_content.cpp

```diff
@@ -591,6 +594,13 @@ void ClientNetworkContentSocketHandler::OnFailure()
 		fclose(this->curFile);
```

Comment:
Should the file be deleted on failure?

Reply:
This wasn't done before, so I didn't dare to touch it. I would think that makes sense, on the other hand, leaving it behind allows you to diagnose errors .. I was a bit meh about it, either way. But not for this PR at least, I would think :)